cmd/abigen/main.go

```diff
@@ -90,6 +96,12 @@ func main() {
 				fmt.Printf("Failed to build Solidity contract: %v\n", err)
 				os.Exit(-1)
 			}
+		} else if *vyFlag != "" {
```

Comment:
turn this into a `switch`

Reply:
sorry not sure how to make this a switch without making it messy! because if/else if are checking conditions on two different variables

Reply:
Go lets you write switches that are more complex than C switches. See https://tour.golang.org/flowcontrol/11